src/views/docs-view/utils/get-root-docs-path-generation-functions.ts

```diff
@@ -32,6 +32,7 @@ export function getRootDocsPathGenerationFunctions(
 		productSlugForLoader: rootDocsPath.productSlugForLoader,
 		basePathForLoader: rootDocsPath.basePathForLoader,
 		mainBranch: rootDocsPath.mainBranch,
+		showVersionSelect: getShowVersionSelect(productData, rootDocsPath),
```

Comment:
We don't want the version select to appear on `/packer/plugins`. Intent here is also to be able to use this same utility for other docs routes.

> **Note**: in some ways, this approach feels a _little_ odd to me, since we could instead set this config up in `page` files and pass it through (rather than `get`-ing it based on `productData` + `rootDocsPath`). At present, since we have separate page files, it feels like we're avoiding a more direct and transparent approach. However, I think this approach will make more sense in a future where we no longer have separate page files for every docs route (which is something we're actively working towards, from what I understand).

Reply:
Agreed on the note here, I think you've set this up well considering additional consolidation efforts. 👍